metric/registry.go

```diff
@@ -37,33 +37,63 @@ const (
 	derivedGaugeFloat64
 )
 
+type metricOptions struct {
```

Comment:
Please leave a TODO here for adding constant labels once the spec is approved.

Reply:
sure.